returnsrcvr-qual/src/main/java/org/checkerframework/checker/builder/qual/ReturnsReceiver.java

```diff
@@ -0,0 +1,24 @@
+package org.checkerframework.checker.builder.qual;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * This declaration annotation indicates that the method on which it is written returns exactly the
+ * receiver object.
+ *
+ * <p>This annotation can only be written on a method declaration. It is inherited by all overrides
+ * of that method.
```

Comment:
Does the new test case also test the override functionality?  Not sure if it matters

Reply:
It does not. I also don't think it really matters, but I can add a test.